packages/fuselage/src/components/Message/MessageMetrics/MessageMetricsFollowing.tsx

```diff
@@ -1,13 +1,29 @@
-import type { ComponentProps } from 'react';
+import type { ComponentProps, ReactElement } from 'react';
 import React from 'react';
 
 import { IconButton } from '../../Button';
 
 type MessageMetricsFollowingProps = {
   name: 'bell' | 'bell-off';
+  badge?: ReactElement;
 } & Omit<ComponentProps<typeof IconButton>, 'icon'>;
 
 export const MessageMetricsFollowing = ({
   name,
+  badge,
   ...props
-}: MessageMetricsFollowingProps) => <IconButton {...props} small icon={name} />;
+}: MessageMetricsFollowingProps) => (
+  <IconButton
+    position='relative'
+    overflow='visible'
+    {...props}
+    small
+    icon={name}
+  >
+    {badge && (
+      <div role='status' className={'rcx-message-metrics__item__follow-badge'}>
```

Comment:
`role='status'` come with `aria-live='polite'` and it will announce this badge, we should handle a better message on it. Maybe attach it to the `IconButton` label